dll/win32/browseui/shellbrowser.cpp

```diff
@@ -786,11 +786,13 @@ HRESULT CShellBrowser::Initialize()
 
     fToolbarProxy.Initialize(m_hWnd, clientBar);
 
-
     // create status bar
-    fStatusBar = CreateWindow(STATUSCLASSNAMEW, NULL, WS_CHILD | WS_VISIBLE | WS_CLIPSIBLINGS |
-                    SBT_NOBORDERS | SBT_TOOLTIPS, 0, 0, 500, 20, m_hWnd, (HMENU)0xa001,
-                    _AtlBaseModule.GetModuleInstance(), 0);
+    DWORD dwStatusStyle = WS_CHILD | WS_CLIPSIBLINGS | SBARS_SIZEGRIP | SBARS_TOOLTIPS;
+    if (fStatusBarVisible)
+        dwStatusStyle |= WS_VISIBLE;
+    fStatusBar = CreateWindowExW(0, STATUSCLASSNAMEW, NULL, dwStatusStyle,
```

Comment:
Unrelated to your actual PR, but using "f" as prefix for fStatusBar is obviously very error-inducing, because this is a window handle that is being returned, not a flag or boolean value. So `hStatusBar` would be more correct there.
(And if this is also a member of the CShellBrowser class, it should have been "m_hStatusBar"... But I think this goes beyond the scope of this PR. This was just my 1-cent rant of the day xD)